src/Query/Filter/SQLFilter.php

```diff
@@ -152,7 +152,14 @@ final public function hasParameter(string $name): bool
      */
     final public function __toString(): string
     {
-        return serialize($this->parameters);
```

Comment:
If we have this – does anybody care about unserializing, and if so, will it put the new `Parameter` class back in place?